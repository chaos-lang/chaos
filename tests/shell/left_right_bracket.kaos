list a = [0]
list f = [1, 2]
a[f] = a
a
f

k[0] = 1
list k = [1, 1, 1, 1]
k
