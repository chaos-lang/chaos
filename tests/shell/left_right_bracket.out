Chaos Error (most recent call last):
File: "~/chaos/__interactive__.kaos", line 3, in <module>
a[f] = a
Unexpected accessor data type: List for variable: a
Absorbed by Interactive Shell
[0]
[1, 2]
Chaos Error (most recent call last):
File: "~/chaos/__interactive__.kaos", line 7, in <module>
k[0] = 1
Undefined variable: k
Absorbed by Interactive Shell
[1, 1, 1, 1]
